libs/foundry-dev-tools/src/foundry_dev_tools/clients/data_health.py

```diff
@@ -0,0 +1,59 @@
+"""Implementation of the data-health API."""
+
+from __future__ import annotations
+
+from typing import TYPE_CHECKING
+
+from foundry_dev_tools.clients.api_client import APIClient
+
+if TYPE_CHECKING:
+    import requests
+
+    from foundry_dev_tools.utils.api_types import (
+        CheckRid,
+        DatasetRid,
+    )
+
+
+class DataHealthClient(APIClient):
```

Comment:
please add new client to https://github.com/emdgroup/foundry-dev-tools/blob/main/libs/foundry-dev-tools/src/foundry_dev_tools/config/context.py#L45